docs/src/main/sphinx/release/release-447.md

```diff
@@ -0,0 +1,47 @@
+# Release 447 (8 May 2024)
+
+## General
+
+* Add support for {doc}`/sql/show-create-function`. ({issue}`21809`)
+* Add support for the {func}`bitwise_xor_agg` aggregation function. ({issue}`21436`)
+* {{breaking}} Require JDK 22.0.0 to run Trino. ({issue}`20980`)
+* Improve performance of `ORDER BY` queries with `LIMIT` on large data sets. ({issue}`21761`)
+* Improve performance of queries containing the {func}`rank` or
+  {func}`row_number` window functions . ({issue}`21639`)
+* Improve performance of correlated queries with `EXISTS`. ({issue}`21422`)
+
+## CLI
+
+* Fix incorrect error location markers for SQL routines causing CLI to print
+  exceptions. ({issue}`21357`)
+
+## Delta Lake connector
+
+* Add support for concurrent `DELETE` pushdown and `TRUNCATE` queries. ({issue}`18521`)
+* Fix under-accounting of memory usage for writing strings to Parquet files. ({issue}`21745`)
+
+## Hive connector
+
+* Add support for the `hive.metastore-cache.cache-missing-stats` and
```

Comment:
This should be reversed "Add support for xxxx feature, which can be enabled via yyyy property". I'm not sure I understand what's the feature here, though (or why did it matter that the table be analyzed to for filesystem caching to work)